### PR TITLE
chore: bump stackable-operator version and cargo update

### PR DESCRIPTION
**Pull request by xeniape**
# Description

PR to fix unmaintained crate cargo deny errors

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes

```[tasklist]
# Author
- [ ] Changes are OpenShift compatible
- [ ] CRD changes approved
- [ ] CRD documentation for all fields, following the [style guide](https://docs.stackable.tech/home/nightly/contributor/docs/style-guide).
- [ ] Helm chart can be installed and deployed operator works
- [ ] Integration tests passed (for non trivial changes)
- [ ] Changes need to be "offline" compatible
```

```[tasklist]
# Reviewer
- [ ] Code contains useful comments
- [ ] Code contains useful logging statements
- [ ] (Integration-)Test cases added
- [ ] Documentation added or updated. Follows the [style guide](https://docs.stackable.tech/home/nightly/contributor/docs/style-guide).
- [ ] Changelog updated
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)
```

```[tasklist]
# Acceptance
- [ ] Feature Tracker has been updated
- [ ] Proper release label has been added
- [ ] [Roadmap](https://github.com/orgs/stackabletech/projects/25/views/1) has been updated
```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,10 @@
 
 - Run a `containerdebug` process in the background of each Airflow container to collect debugging information ([#557]).
 
+### Changed
+
+- Bump `stackable-operator` from `0.82.0` to `0.83.0` ([#565]).
```
Comment (sbernauer) [inline CHANGELOG.md:11]: We normally don't document op-rs bumps in here - we only do so when there are user-facing changes

**Review by sbernauer — CHANGES_REQUESTED** [review]

**Comment by sbernauer** [comment]
Thanks for fixing them!

File: CHANGELOG.md (hunk 1 above)
Comment (xeniape) [reply CHANGELOG.md]: https://github.com/stackabletech/airflow-operator/pull/565/commits/f4c49985b03e61c672ab2f59b7a8c8aa7281a810

**Review by sbernauer — APPROVED** [review]
Thanks!

**Pull request merged**